infra/ansible/setup-worker.yml

```diff
@@ -0,0 +1,53 @@
+---
+- hosts: workers:&local
+  become: yes
+  gather_facts: yes
+  tasks:
+    - name: Get hosts
+      debug: var=ansible_play_hosts
+    - name: Get hostname
+      debug: var=ansible_facts.hostname
+    - name: Add keys to authorized_keys
+      authorized_key:
+        user: ubuntu
+        state: present
+        key: http://172.16.0.10/authorized_keys
+    - name: get filebeat.yml from http server on manager
+      get_url:
+        url: http://172.16.0.10/filebeat.yml
```

Comment:
Similar question as the one above - why do we need to fetch the `filebeat.yml` from the `master` and expose over HTTP ? I think this file should be static, and we should be able to apply it to `/etc/filebeat/filebeat.yml` directly with Ansible.

Reply:
Alternatively, I could have used the same strategy that I use on the manager node where I build the filebeat.yml from a template using the secrets stored in the S3 bucket.